cmd/plume/prerelease.go

```diff
@@ -475,7 +476,27 @@ func azurePreRelease(ctx context.Context, system string, client *http.Client, sr
 	return nil
 }
 
-func awsUploadToPartition(spec *channelSpec, part *awsPartitionSpec, imageName, imageDescription, imagePath string) (map[string]string, map[string]string, error) {
+func getSpecAWSImageName(spec *channelSpec, imageMetadata *map[string]interface{}) (string, string, string) {
+	imageFileName := spec.AWS.Image
+	imageName := fmt.Sprintf("%v-%v-%v", spec.AWS.BaseName, specChannel, specVersion)
+	imageName = regexp.MustCompile(`[^A-Za-z0-9()\\./_-]`).ReplaceAllLiteralString(imageName, "_")
```

Comment:
Is it expected that `\` & `/` are allowed characters? And will `/` not conflict with the s3 buckets?

Reply:
This is something which was already there, and I just moved it into a util method.

Reply:
cc @bgilbert 

Is there a potential bug if images were to have `/` in the file name that would dump it into a new bucket?

Reply:
In the original code, at least, the `imageName` is used for the human-readable snapshot and AMI names but _not_ for the S3 object name.